src/main/java/teammates/ui/template/FeedbackSessionsList.java

```diff
@@ -0,0 +1,18 @@
+package teammates.ui.template;
+
+import java.util.List;
+
+public class FeedbackSessionsList {
+    
+    private List<FeedbackSessionRow> existingFeedbackSessions;
+    
+    public FeedbackSessionsList(List<FeedbackSessionRow> existingFeedbackSessions) {
+        this.existingFeedbackSessions = existingFeedbackSessions;
+    }
```

Comment:
Is there a reason for creating this class instead of simply using a list of FeedbackSessionRow?